### PR TITLE
Remove other countries from build folder

### PR DESCRIPTION
**Pull request by PysarenkoDS-BWT**
Rewrite build script for deleting unused countries from build

Script get selected script from `env` - `REACT_APP_COUNTRY` and then delete other countries, if country not providing do nothing

### CODE REVIEW

File: config-overrides.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const {
+  override,
+  disableChunk,
+  useEslintRc,
+  addWebpackPlugin,
+} = require('customize-cra');
+const path = require('path');
+const RemovePlugin = require('remove-files-webpack-plugin');
+
+const country = process.env.REACT_APP_COUNTRY;
```
Comment (ericboucher) [inline config-overrides.js:10]: Should we take the lowercase version to be safe?

Comment (ericboucher) [inline config-overrides.js:10]: Should we take the lowercase version?
Also, you need to have a default country, or actually, do not filter the data folder if no country is specified via this environment variable

Comment (PysarenkoDS-BWT) [reply config-overrides.js:10]: I type it in description, it's already implemented on 15th line, if environment variable not provided RemovePlugin not added

Comment (ericboucher) [reply config-overrides.js:10]: perfect, thanks

**Review by ericboucher — APPROVED** [review]

**Pull request merged**